code/game/objects/machinery/overwatch.dm

```diff
@@ -712,20 +764,108 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 	ID.assigned_fireteam = 0 //reset fireteam assignment
 
 	//Changes headset frequency to match new squad
-	var/obj/item/radio/headset/mainship/marine/H = transfer_marine.wear_ear
-	if(istype(H, /obj/item/radio/headset/mainship/marine))
-		H.set_frequency(new_squad.radio_freq)
+	var/obj/item/radio/headset/mainship/marine/headset = transfer_marine.wear_ear
+	if(istype(headset, /obj/item/radio/headset/mainship/marine))
+		headset.set_frequency(new_squad.radio_freq)
 
 	transfer_marine.hud_set_job()
-	if(issilicon(usr))
-		to_chat(usr, span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
+	if(issilicon(source))
+		to_chat(source, span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
 	visible_message(span_boldnotice("[transfer_marine] has been transfered from squad '[old_squad]' to squad '[new_squad]'. Logging to enlistment file."))
 	to_chat(transfer_marine, "[icon2html(src, transfer_marine)] <font size='3' color='blue'><B>\[Overwatch\]:</b> You've been transfered to [new_squad]!</font>")
 
-///This is an orbital light. Basically, huge thing which the CIC can use to light up areas for a bit of time.
-/obj/machinery/computer/camera_advanced/overwatch/proc/attempt_spotlight(datum/source, atom/A, params)
+
+///Signal handler for radial menu
+/obj/machinery/computer/camera_advanced/overwatch/proc/attempt_radial(datum/source, atom/A, params)
 	SIGNAL_HANDLER
+	INVOKE_ASYNC(src, PROC_REF(do_radial), source, A, params)
+
+///Quick-select radial menu for Overwatch
+/obj/machinery/computer/camera_advanced/overwatch/proc/do_radial(datum/source, atom/A, params)
+	var/list/radial_options
```

Comment:
Can just cut this var entirely since you can slot in the appropriate list for each show_radial_menu call. Aside from that, have you done a last retest to make sure everything works the way it should?

Reply:
Everything should work fine but additional testing would still probably be a good idea as a few bugs have probably still snuck in

Reply:
done